packages/wrangler/scripts/bundle.ts

```diff
@@ -1,26 +1,14 @@
 import fs from "node:fs/promises";
 import path from "node:path";
-import { build } from "esbuild";
+import { build, context } from "esbuild";
 import { EXTERNAL_DEPENDENCIES } from "./deps";
-import type { WatchMode } from "esbuild";
+import type { Format, Platform } from "esbuild";
 
 // the expectation is that this is being run from the project root
 type BuildFlags = {
 	watch?: boolean;
 };
 
-function watchLogger(outputPath: string): WatchMode {
-	return {
-		onRebuild(error, _) {
-			if (error) {
-				console.error(`${outputPath} build failed.\n`, error);
```

Comment:
Do we lose these log messages?

Reply:
[This] (https://github.com/cloudflare/workers-sdk/pull/3772/files#diff-08737a62058e3e6d5614a637f6cbdf460b6238a2f749bc2d30de3756f8463e51R116) should cover errors on rebuild